go.mod

```diff
@@ -5,23 +5,24 @@ go 1.12
 require (
 	github.com/BurntSushi/toml v0.2.0
 	github.com/IBM-Cloud/ibm-cloud-cli-sdk v0.2.0
-	github.com/davecgh/go-spew v1.1.0
+	github.com/davecgh/go-spew v1.1.1
```

Comment:
Is there any new dependency added hence mod and sum file needed modification ? if not then lets not update these files. I did not see any new dependency

Reply:
I did not modify these files intentionally. These dependency gets updated automatically if newer version available while building the code.

Reply:
Reverted back to old file.